### PR TITLE
Added How to create and use materialized views guide

### PR DESCRIPTION
**Pull request by ayush2390**

### CODE REVIEW

**Comment by ayush2390** [comment]
@aijamalnk @ericankenman I have added the **How to create and use materialized views** guide

File: docs/postgres_guides/how-to-create-and-use-materialized-views/how-to-create-and-use-materialized-views.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+**Step 2** - Create a materialized view using the `CREATE MATERIALIZED VIEW` statement. Here’s an example to create a materialized view:
+
+```
+CREATE MATERIALIZED VIEW customer_order_totals AS
+SELECT customer_id, SUM(order_amount) AS total_amount
+FROM orders
+GROUP BY customer_id;
+```
+
+In this example-
```
Comment (rzavalet) [inline docs/postgres_guides/how-to-create-and-use-materialized-views/how-to-create-and-use-materialized-views.md:30]: In this example:

File: docs/postgres_guides/how-to-create-and-use-materialized-views/how-to-create-and-use-materialized-views.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+- `customer_order_totals` is the name of the view
+- `customer_id` and `order_amount` are the column whose data we want to store in created materialized view
```
Comment (rzavalet) [inline docs/postgres_guides/how-to-create-and-use-materialized-views/how-to-create-and-use-materialized-views.md:33]: are the columns

in the created

File: docs/postgres_guides/how-to-create-and-use-materialized-views/how-to-create-and-use-materialized-views.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+- `orders` is the name of the table
+- `customer_id` is the condition too filter out the data
```
Comment (rzavalet) [inline docs/postgres_guides/how-to-create-and-use-materialized-views/how-to-create-and-use-materialized-views.md:35]: too -> to

**Review by rzavalet — COMMENTED** [review]
Some minor comments

File: docs/postgres_guides/how-to-create-and-use-materialized-views/how-to-create-and-use-materialized-views.md (hunk 3 above)
Comment (ayush2390) [reply docs/postgres_guides/how-to-create-and-use-materialized-views/how-to-create-and-use-materialized-views.md]: Addressed the comments

Comment (rzavalet) [reply docs/postgres_guides/how-to-create-and-use-materialized-views/how-to-create-and-use-materialized-views.md]: Looks good to me!

**Review by rzavalet — APPROVED** [review]

**Review by ericankenman — APPROVED** [review]

**Pull request merged**